src/components/SidePanes/UsersPane.jsx

```diff
@@ -47,6 +49,13 @@ class UsersPane extends React.Component {
     });
   }
 
+  cancelAddUser() {
+    this.setState({
+      addingUser: false,
+      newEditorChoice: 0,
```

Comment:
What does setting this to `0` mean? Like, I know it is supposed to be a user id, but is there a chance we are going to send this 0 anywhere? If so, what is the impact of that? Is it better to send `undefined` or `null`?